dashboard/app/entities.go

```diff
@@ -123,6 +123,21 @@ type Bug struct {
 	Labels         []BugLabel
 	DiscussionInfo []BugDiscussionInfo
 	TreeTests      BugTreeTestInfo
+	FixCandidates  []BugFixCandidate
+}
+
+type FixCandidateSource string
+
+const (
+	BisectionFixCandidate FixCandidateSource = "bisection"
+)
+
+type BugFixCandidate struct {
+	Source FixCandidateSource
+	Repo   string
+	Branch string
+	Commit Commit
```

Comment:
If we show this anywhere, we will need Title and maybe other info.
This should probably include Commit object. We already should have some to render Commit objects.

Reply:
No longer relevant as the code is deleted.